app/src/main/java/com/youssef/fixit/Auth/LoginFragment/LoginPresenter.java

```diff
@@ -40,6 +41,7 @@ void Validation(String Mail, String Password) {
     }
 
     void LoginRequest(String Mail, String Password) {
```

Comment:
Method naming convention should start with lower case 
check out this [link](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Methods%20should%20be%20verbs%2C%20in,of%20each%20internal%20word%20capitalized.&text=Except%20for%20variables%2C%20all%20instance,words%20start%20with%20capital%20letters.) about conventions